src/formaction.cpp

```diff
@@ -117,23 +117,19 @@ bool FormAction::process_op(Operation op,
 		break;
 	case OP_INT_SET:
 		if (automatic) {
-			std::string cmdline = "set ";
-			if (args) {
-				for (const auto& arg : *args) {
-					cmdline.append(strprintf::fmt(
-							"%s ", Stfl::quote(arg)));
-				}
+			if (args && args->size() == 2) {
+				const std::string key = args->at(0);
+				const std::string value = args->at(1);
+				cfg->set_configvalue(key, value);
+				return true;
+			} else {
+				v->show_error(_("usage: set <config-option> <value>"));
```

Comment:
I think it'd be better for consistency if we reused the existing message here, which says `usage: set <variable>[=<value>]`.

Reply:
That way of using it is not supported in macros (it wasn't supported before either, it is only supported on the `cmdline`).

I think supporting it might not be all that easy.
Take for example the following lines:
```
macro a set browser "firefox --new-window %u" ; open-in-browser # Currently supported
macro b set "browser=firefox --new-window %u" ; open-in-browser # Easy to support (probably)
macro c set browser="firefox --new-window %u" ; open-in-browser # Does not work well with `parse_operation_sequence()` (probably)
```

The `macro c` line looks nice to me but I believe it will take quite some work to parse it correctly.
Currently, the parser splits it up in:
```
macro
set
browser="firefox
--new-window
%u"
```
which is not sufficient for properly implementing this.

Reply:
Oh, my bad! You're totally right. Thanks for the explanation!